server/dba/migrations/46-addForestCords.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
Would you mind running a formatter on this? If you don't already have anything, you can use prettier's defaults:

```
npx prettier server/deva/migrations/46-addForestCords.js -w
```